typescript/packages/plugins/erc721/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# @goat-sdk/plugin-erc20
```

Comment:
No need to add this, it will be generated automatically by changesets :)

Reply:
ok removed